### DISCUSSION TITLE
"Ok, here we go..." - how to get the full answer?

### DISCUSSION THREAD

**Discussion by Ether90** [oobabooga/text-generation-webui#247] [General]
Hey, I just started today. I use the 30b-4 model.

All is working really good, but sometimes I request a longer response, something like "write a story" or something and many times I get "Ok, here we go..." and nothing more. The funny part is that the bot ALWAYS uses exactly that phrase: "Ok, here we go...", but nothing more.

Are there some tricks to trigger longer answers? I played with "min_length", but this just outputs things like that:

`Yourself Is Through Travel | View comments » The Best Way To Learn About Yourself Is Through Travel | View comments » The Best Way To Learn About Yourself Is Through Travel | View comments » The Best Way To Learn About Yourself Is Through Travel | `

after the normal answer.

Cheers

**Comment by oobabooga** [comment]
Try increasing `Generation attempts (for longer replies)` under `Generation settings` in chat mode. Or just reply with the following, it usually works:

> Go on...
